.github/workflows/release.yml

```
@@ -0,0 +1,626 @@
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          # Use the lowest supported version of Python for CI/CD
+          python-version: "3.9"
+      - name: Load cached .local
+        id: cache-poetry
+        uses: actions/cache@v3
```

actionlint (1.7.8)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)